tensorbay/cli/log.py

```diff
@@ -354,9 +354,15 @@ def _get_merge_prefixes(self) -> Iterator[str]:
         for color in self._layer_colors:
             prefixes.append(click.style("|", fg=color))
             prefixes.append(" ")
+        color_number = len(self._layer_colors)
         for i in range(self._merge_pointer, self._pointer, -1):  # type: ignore[arg-type]
             temp_prefixes = prefixes.copy()
             temp_prefixes[2 * i - 1] = click.style("/", fg=self._layer_colors[self._pointer])
+            for j in range(color_number - self._merge_pointer):  # type: ignore[operator]
```

Comment:
where does this mypy error come from？

Reply:
`self._merge_pointer` is `Optional[int]`, it may be `None`

Reply:
set `self._merge_pointer` as an argument?